include/z64.h

```diff
@@ -1765,4 +1765,10 @@ typedef struct {
     /* 0x20 */ u16 fracPart[4][4];
 } MatrixInternal; // size = 0x40
 
+typedef struct {
+    /* 0x00 */ u8* value;
+    /* 0x04 */ const char* name;
+} FlagSetEntry; // size = 0x08
+
+
```

Comment:
You added an extra line here
```suggestion
} FlagSetEntry; // size = 0x08

```